crates/amaru-ledger/src/context/assert.rs

```diff
@@ -252,6 +269,14 @@ impl WitnessSlice for AssertValidationContext {
         }
     }
 
+    #[instrument(
+        level = Level::TRACE,
+        fields(
+            bootstrap_witness.hash = %root,
+        )
+        skip_all,
+        name = "require_bootstrap_witness"
```

Comment:
Note: I don't think you need to specify the name when it's identical to the function being instrumented.

Reply:
You may be right, but I like specifying it -- in case something changes or whatever